config/rubocop.yml

```diff
@@ -110,8 +111,8 @@ Metrics/BlockLength:
     - "spec/**/*.rb"
     - "config/**/*.rb"
 
-Metrics/LineLength:
-  Max: 120
+Layout/LineLength:
+  Max: 130
```

Comment:
120 was a good value, why change?

Reply:
That change was introduced by me because in one of our Ragnarson-generic recipes line length was bigger then 120, but I didn't thought about breaking lines to make it fit rubocop requirements. 
Im going to revert that change, back to 120.

Reply:
@michalgoldys 120 column length rule is an industry-standard and we'd like to comply with it.

Reply:
You can find more info in cop description - https://rubystyle.guide/#max-line-length

Reply:
@stuszynski Yeah, you're right. I've already made a change to that file ;) I won't mess with it again. Sorry for hearth attack :)